wrappers/ds_selinux_restorecon.sh.in

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# BEGIN COPYRIGHT BLOCK
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# END COPYRIGHT BLOCK
+
+# Make sure we have the path to the dse.ldif
+if [ -z $1 ]
+then
+    echo "usage: ${0} /etc/dirsrv/slapd-<instance>/dse.ldif"
+    exit 1
+fi
+
+# Grep the db_home_dir out of the config file
+DS_HOME_DIR=`grep 'nsslapd-db-home-directory: ' $1 | awk '{print $2}'`
+
+# Now run restorecon
+restorecon ${DS_HOME_DIR}
```

Comment:
Should use ${DS_HOME_DIR:?} or explicitly test that DS_HOME_DIR is not empty to avoid running restorecon on the whole system if dse file is broken or deleted

Reply:
Thanks Pierre for the review, this is now fixed.